.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python: python3.6
+            dist: ubuntu:bionic
+          - python: python3.8
+            dist: ubuntu:focal
+          #- python: python3.10
+          #  dist: ubuntu:jammy
+    steps:
+      - uses: actions/checkout@v3
+      - name: Pull docker container
+        run: docker pull ${{ matrix.dist }}
```

Comment:
Is there really a good reason to use Docker manually at this point? If I remember correctly, that was mainly a Travis CI thing? Couldn't we just use `runs-on: ${{ matrix.dist }}` above and run the script directly instead (probably after running [setup-python](https://github.com/actions/setup-python)).

Btw, to try those things out locally, there's [act](https://github.com/nektos/act), in case you weren't aware of it yet.